src/main/scala/ch/openolitor/arbeitseinsatz/eventsourcing/ArbeitseinsatzEventStoreSerializer.scala

```diff
@@ -26,14 +26,21 @@ import ch.openolitor.arbeitseinsatz._
 import ch.openolitor.arbeitseinsatz.ArbeitseinsatzCommandHandler.SendEmailToArbeitsangebotPersonenEvent
 import ch.openolitor.arbeitseinsatz.models._
 import ch.openolitor.core.JSONSerializable
+import ch.openolitor.core.Macros.copyTo
 import ch.openolitor.core.domain.EntityStoreJsonProtocol
-import ch.openolitor.stammdaten.models.PersonContactPermissionModify
+import ch.openolitor.core.eventsourcing.CoreEventStoreSerializer
+import ch.openolitor.mailtemplates.eventsourcing.MailTemplateEventStoreSerializer
+import ch.openolitor.stammdaten.StammdatenJsonProtocol
+import ch.openolitor.stammdaten.models.{ Person, PersonContactPermissionModify }
+import spray.json.JsValue
 import spray.json.lenses.JsonLenses._
-import stamina.{ V1, V2 }
+import stamina.{ V1, V2, V3 }
```

Comment:
I guess you should run organize imports to reduce number of warnings?

Reply:
I have removed the ones that are not used right now. If you have added the "organize import" in your otp story, we can integrate it when merging these branch